samples/instrumentation-quickstart/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.12
+WORKDIR /usr/src/app
+COPY . .
+RUN python -m venv /venv && \
+    /venv/bin/pip install -r requirements.txt
+CMD bash -c "source /venv/bin/activate && flask run --host 0.0.0.0 --port 8080 2>&1 | tee /var/log/app.log"
```

Comment:
When i was working on the [JS sample](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/v2.2.0/v0.18.0/samples/instrumentation-quickstart/Dockerfile), I realized the `bash -c` is not actually needed since docker uses a real shell for CMD

Reply:
Removed.  I had to change "source" to "." because of the change from `bash` to `sh`.